src/ClientOptionsTrait.php

```diff
@@ -115,6 +118,22 @@ private function buildClientOptions(array $options)
         // Keep track of the API Endpoint
         $apiEndpoint = $options['apiEndpoint'] ?? null;
 
+         // If the user provides a logger or the GOOGLE_SDK_DEBUG_LOGGING environment variable is set
+        // we attach a logger to the options array
+        // If we have the logger option SET to null, it should trump the environment variable AKA explicitely
+        // turning off logging
+        if (getenv('GOOGLE_SDK_DEBUG_LOGGING') ||
```

Comment:
This should check the value of `GOOGLE_SDK_DEBUG_LOGGING`. The way it is now, if it was set to `false`, or `bamboozle`, it would still evaluate to true. Better to make this something like

```php
if ('1' === getenv('GOOGLE_SDK_DEBUG_LOGGING')
```

Reply:
Ahhh is going to be interpreted as a `string`! You're right, let me fix that.